tietokanta/src/main/resources/db/migration/V1_436__.sql

```diff
@@ -0,0 +1,15 @@
+UPDATE sanktio
+SET sakkoryhma = 'muistutus' :: sanktiolaji
+WHERE maara IS NULL;
+
+UPDATE sanktio
+SET maara = NULL
+WHERE sakkoryhma = 'muistutus' :: sanktiolaji;
+
+DELETE FROM sanktio WHERE sakkoryhma IS NULL;
```

Comment:
Jaa keskustelu ei toiminut ongelmitta kun tiedoston nimi muuttui. Näitä on tuotannossa 0, kuten kommentoin alla.